From febba068965b69c8fed13f3cae528197f9e89e38 Mon Sep 17 00:00:00 2001
From: dhacker29 <dhackerdvm@gmail.com>
Date: Sat, 26 Apr 2014 22:39:02 -0400
Subject: [PATCH] Don't use MSM_BSP for msm7x27a

Change-Id: I83d680cec2f88644e4d9267208008157f6d7941d
---
 minui/Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/minui/Android.mk b/minui/Android.mk
index 2d3198a..6436f5f 100644
--- a/minui/Android.mk
+++ b/minui/Android.mk
@@ -17,9 +17,11 @@ ifeq ($(call is-vendor-board-platform,QCOM),true)
   LOCAL_C_INCLUDES += $(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include
 endif
 
+ifneq ($(TARGET_BOARD_PLATFORM),msm7x27a)
 ifeq ($(TARGET_USES_QCOM_BSP), true)
     LOCAL_CFLAGS += -DMSM_BSP
 endif
+endif
 
 LOCAL_MODULE := libminui
 
-- 
1.9.1

